README.md

```diff
@@ -126,18 +126,21 @@ Route::get('/forums', 'FooController@bar')
 ```
 
 - ### Custom callbacks
-The callbacks above might not be enough for your own use case, so you might want to add custom callbacks, the following guide will describe the steps to create your own custom callbacks.
+The callbacks above might not be enough for your own use case, so you might want to add custom callbacks, in this guide we will go through several methods to define custom callbacks.
 
-1. Create a new class, for instance `CustomCallbacks`
-2. Add as many callbacks as you want to add, but be sure that all of your methods are **`static`** or you'll be facing problems
-3. Open the `config/geo-routes.php` configuration file, and add your callbacks to the callbacks array, like so:
+ #### 1. Using a class
+
+  1. Create a new class, for instance `CustomCallbacks`
+  2. Add as many callbacks as you want to add, but be sure that all of your methods are **`static`** or you'll be facing problems
```

Comment:
Actually yes. Because configuration callbacks are being loaded using the [`addCallback`](https://github.com/LaraCrafts/laravel-geo-routes/blob/b5211c53e22d51e40a1d80a986b301284b87d047/src/CallbacksRegistrar.php#L35) method which expects a *callable* for its second argument. The [`parseCallbacks`](https://github.com/LaraCrafts/laravel-geo-routes/blob/b5211c53e22d51e40a1d80a986b301284b87d047/src/CallbacksRegistrar.php#L60) method uses [*Reflection*](https://www.php.net/manual/en/book.reflection.php) and can ignore the fact of whether a method is `static` or not, but not to cause confusion and have to make more checks, i thought it is better to only accept `static` methods. 
The [`getClosure`](https://www.php.net/manual/en/reflectionmethod.getclosure.php) method expects an `$object` argument if the method is not `static` and it also results in an error if the `$object` argument is present when the method is static, and that will result us in checking whether the method is `static` or *non-static* which honestly i think of as not necessary at all, so it's as simple as it is, whenever using a class to define your callbacks your methods **must** be `static`.

Reply:
I would do something like `App::call('App\SomeClass@method')`, this way the users of this functionality can also rely on dependency injection.

Reply:
I think this should be implemented in a different pull request, as it is not really the topic of the current one.